packages/fscomponents/src/components/MultiCarousel/MultiCarousel.web.tsx

```diff
@@ -281,9 +281,13 @@ export const MultiCarousel = <ItemT, >(props: MultiCarouselProps<ItemT>) => {
   );
 
   const goToNext = useCallback(async (options?: GoToOptions | GestureResponderEvent) => {
-    const nextIndex = currentIndex + 1 > numberOfPages - 1 ? numberOfPages - 1 : currentIndex + 1;
+    const nextIndex = currentIndex + 1 > numberOfPages - 1 ? 0 : currentIndex + 1;
+    const animationOptions =
```

Comment:
All my same comments from #2324 apply. 

https://github.com/brandingbrand/flagship/pull/2324

Reply:
Cleaned up this code with @ckornell , and removed the need for this line